policies/lib/all_attestations_test.rego

```diff
@@ -0,0 +1,6 @@
+package lib
+
+test_all_attestations {
+	expected := [{"data": {"foo": 10}}]
+	expected == all_attestations with data.attestations as {"foo": 10}
```

Comment:
And here, it indicates data.attestations is singular. So maybe it should be called "attestation"..? Which then makes me unsure if we want to support a list of them, and maybe in that case all_attestations is no longer needed.

(I might be missing something.)

Reply:
Sounds good to me.

Reply:
A cloudy memory returns to me: https://github.com/redhat-appstudio/infra-deployments/blob/main/hack/chains/kaniko-cosign-verify.sh#L109 .

So I think the jq magic would need to deal with "{..att1..}\n{..att2.}\n". maybe that's JSONL format?